endpointmanager/pkg/endpointmanager/postgresql/store_test.go

```diff
@@ -38,9 +38,6 @@ func setupConfig() {
 
 	viper.SetDefault("dbhost", "localhost")
 	viper.SetDefault("dbport", 5432)
-	viper.SetDefault("dbuser", "postgres")
-	viper.SetDefault("dbpass", "postgrespassword")
-	viper.SetDefault("dbname", "postgres")
```

Comment:
why not replace w `lantern`?

Reply:
This whole function is going to get deleted once your integration test PR is merged in